tests/github/fair_software/badge/test_checker_with_noreason_user_config.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from requests_mock import Mocker
+from howfairis import Checker
+from howfairis import Repo
+from tests.contracts.checker import Contract
+
+
+def get_mocked_checker(user_config_filename):
+    repo = Repo("https://github.com/fair-software/badge")
+    return Checker(repo, user_config_filename=user_config_filename)
+
+
+@pytest.fixture
+def user_config(tmp_path):
+    config = tmp_path / "howfairis.yml"
+    text = """skip_repository_checks_reason: ""
+skip_license_checks_reason: ""
+skip_registry_checks_reason: ""
+skip_citation_checks_reason: ""
+skip_checklist_checks_reason: ""
+"""
+    config.write_text(text)
+    yield config
+
+
+class TestCheckerWithReasonableUserConfig(Contract):
```

Comment:
BTW How would you feel about using `TestChecker` in all of these tests? In my experience, the class name here tends to be out of sync with the containing file name.

Reply:
we can discuss in later iterations of writing tests, good for now